src/lib/containers/widgets/ImageButtonWithTooltip.tsx

```diff
@@ -11,40 +11,57 @@ type CustomImageProps = {
 }
 
 type ImageButtonWithTooltipProps = {
```

Comment:
The change in this component allows for a string value to be a trigger - not just an image.